pip/utils/__init__.py

```diff
@@ -3,7 +3,9 @@
 import contextlib
 import errno
 import locale
-import logging
+# we have a submodule named 'logging' which would shadow this if we used the
+# regular name:
+import logging as std_logging
```

Comment:
OK, yeah, after a quick skim, it looks like this isn't technically needed anymore. At some point during development the `logger.getEffectiveLevel() <= logging.INFO` check was happening inside call_subprocess, and erroring out because logging.INFO didn't exist, which confused the heck out of me for a while. So I think this renaming is still a good change, though now technically unrelated to the rest of the PR.